app/assets/javascripts/discourse/components/edit-category-security.js.es6

```diff
@@ -13,6 +13,15 @@ export default buildCategoryPanel("security", {
       }
     },
 
+    onDropdownChange() {
+      Ember.run.later(() => {
```

Comment:
I'm not comfortable with this hack, but I had to do it because `selectedGroup` is mutated after `onSelect` is triggered, and I need the new value of  `selectedGroup`. I could use an observer here, but wanted to check first if there is a better way.

Reply:
Does `Ember.run.scheduleOnce('afterRender', () => ... )` work? I really am against having delays like this in our codebase. cc @jjaffeux

Reply:
That was the first thing I tried (and I retried again just to make sure), but sadly it didn't work.
I know you don't like observers, but for now I've switched to an observer because I think it's much better than a delay.